Workbooks/WorkbooksMetadata.json

```diff
@@ -945,6 +945,32 @@
     "subtitle": "",
     "provider": "Citrix Systems Inc."
   },
+  {
+    "workbookKey": "UnifiSGWorkbook",
+    "logoFileName": "UnifiSG.svg",
+    "description": "Gain insights into Unifi Security Gateways analyzing traffic and activities.",
+    "dataTypesDependencies": [ "CommonSecurityLog" ],
+    "dataConnectorsDependencies": ["UnifiSG"],
+    "previewImagesFileNames": [ "UnifiSGBlack.png", "UnifiSGWhite.png" ],
+    "version": "1.0",
+    "title": "Unifi Security Gateway",
+    "templateRelativePath": "UnfiSG.json",
+    "subtitle": "",
+    "provider": "Azure Sentinel community"
+  },
+  {
+    "workbookKey": "UnifiSGNetflowWorkbook",
+    "logoFileName": "UnifiSG.svg",
+    "description": "Gain insights into Unifi Security Gateways analyzing traffic and activities using Netflow.",
+    "dataTypesDependencies": [ "netflow_CL" ],
```

Comment:
please update datatype in workbook metadata and workbooks too.

Reply:
this is generic netflow.  unifi uses the industry standard netflow formats.